lib/ex_admin/ex_admin.ex

```diff
@@ -237,7 +210,7 @@ defmodule ExAdmin do
 
   @doc false
   def get_title_actions(%Plug.Conn{private: _private, path_info: path_info} = conn) do
-    defn = get_registered_by_controller_route(path_info)
+    defn = conn.assigns.defn
```

Comment:
@smpallen99 What is the purpose of namespaces here? We assign these keys to connections that handle requests to admin controllers. So, no interference with frontend-controllers is possible.
As to `ea_required`, maybe `required_fields` would be more descriptive?